### PR TITLE
Correct OTA build

### PR DESCRIPTION
**Pull request by pablomarinoacn**

### CODE REVIEW

File: .github/workflows/WhenMainChange.yml [hunk 1]
```diff
@@ -2,12 +2,21 @@ name: Build Android
 run-name: Creating Android builds
 
 on:
+  #Delete this , just for test
```
Comment (andreslopezsarmiento) [inline .github/workflows/WhenMainChange.yml:5]: delete this

File: .github/actions/buildAndUpload/action.yaml [hunk 2]
```diff
@@ -47,9 +48,10 @@ runs:
         api_key=${{ inputs.Build-OTA-APIKEY }}
         api_key_build=${{ inputs.Build-OTA-APIKEY-BUILD }}
         visibleClient=true
-        buildType="upper(${{ inputs.Build-ENVL }})"
-        name="Esmorga [upper(${{ inputs.Build-ENVL }})]"
-        
+        visibleClient=true
```
Comment (andreslopezsarmiento) [inline .github/actions/buildAndUpload/action.yaml:51]: is this line duplicated?

**Review by andreslopezsarmiento — CHANGES_REQUESTED** [review]

File: .github/workflows/WhenMainChange.yml (hunk 1 above)
Comment (pablomarinoacn) [reply .github/workflows/WhenMainChange.yml:5]: 👍

File: .github/actions/buildAndUpload/action.yaml (hunk 2 above)
Comment (pablomarinoacn) [reply .github/actions/buildAndUpload/action.yaml:51]: fuck

**Review by andreslopezsarmiento — APPROVED** [review]

**Pull request merged**